src/main/java/org/worldcubeassociation/ui/ExportJSONAction.java

```diff
@@ -45,7 +46,7 @@ public void actionPerformed(ActionEvent aActionEvent) {
         }
 
         try {
-            String jsonFileName = "Results for " + getEnv().getCompetitionId() + ".json";
+            String jsonFileName = "Results (" + WorkbookAssistant.class.getPackage().getImplementationVersion() + ") for " + getEnv().getCompetitionId() + ".json";
```

Comment:
We can certainly show this new field on the WCA results uploading page.

Given a JSON file, how will you know which program it came from?

CCM is meant to be downloaded and run offline, so it could suffer from the same problems as the WA. Even with cubecomps, results from a year ago were using a version of cubecomps that is not up to date =)